src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -214,11 +218,26 @@ void updateStageUser(UUID caseUUID, UUID stageUUID, UUID newUserUUID) {
     Set<Stage> getActiveStagesByTeamUUID(UUID teamUUID) {
         log.debug("Getting Active Stages for Team: {}", teamUUID);
         Set<Stage> stages = stageRepository.findAllActiveByTeamUUID(teamUUID);
+        addSomuData(stages);
```

Comment:
I'm a little confused why on a get we are adding data, so probably worth some comments?

Reply:
I've added a comment but to clarify here aswell the front end calls this for its information when loading the team dashboards and it returns information from the case_data table, the data field in that table contains unreliable data for the case contributions. So to add contribution statuses I needed the reliable data from the somu_item table so I've added it to the data that is sent.

Reply:
Awesome, thanks for explaining 👍🏽